pkgs/development/python-modules/bitcoinlib/default.nix

```diff
@@ -15,7 +15,7 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace bitcoin/core/key.py --replace \
-      "ctypes.util.find_library('ssl') or 'libeay32'" \
+      "ctypes.util.find_library('ssl.35') or ctypes.util.find_library('ssl') or 'libeay32'" \
```

Comment:
This change isn't really needed to make the package build again but since the code was updated it felt right to just replace the entire line with the proper path to libssl.